templates/component.html

```diff
@@ -5,12 +5,11 @@
     <div class="content-wrapper">
         <div class="hxRow">
             <div class="hxCol-8">
-                <div class="hxRow">
-                    <div class="hxCol">
-                        <h1 class="hxPageTitle">{{page.title}}</h1>
-                        <div class="component-blurb">{{page.usage}}</div>
-                    </div>
+                <div class="inline-block">
```

Comment:
Agreed. Should we address this in this PR or merge and look as part of the larger 'make the site responsive' issue?

Reply:
This can probably be handled with the current responsive classes in Helix, by tinkering with the column sizing classes, and placing them on the hxCols in the header.

You could consider setting the thumbnail and title columns to 12 at smaller screen sizes, so they stack, instead of wrapping.